packages/beacon-node/package.json

```diff
@@ -124,7 +124,6 @@
     "@lodestar/db": "^1.9.1",
     "@lodestar/fork-choice": "^1.9.1",
     "@lodestar/light-client": "^1.9.1",
-    "@lodestar/logger": "^1.9.1",
```

Comment:
@lodestar/logger should only be a dependency of @lodestar/cli, other packages use dependency injection to get access to logger (could be different implementation)